internal/cmd/edit_test.go

```diff
@@ -26,24 +28,30 @@ import (
 	"github.com/google/go-containerregistry/pkg/v1/random"
 )
 
-func TestEditConfig(t *testing.T) {
-	src := "gcr.io/crane/edit/config"
-	reg, err := registry.TLS("gcr.io")
+func mustRegistry(t *testing.T) (*httptest.Server, string) {
```

Comment:
This seems like a useful utility to share in an exported `internal` package somewhere. Maybe even an exported non-`internal` one tbh.

Reply:
If we make it reusable publicly we could try to make it return `host, close` so folks can just `defer close()` and not have visibility into the registry at all besides the host.